fluid/example/slicing/linked-outputs/bar-chart-line-chart-2.fld

```diff
@@ -0,0 +1,29 @@
+MultiView {
```

Comment:
This might be overkill for the new test – would it not make more sense just to move one of the test files from `fluid/example/..` to `test/fluid/..` as we discussed earlier? That prototypes the structure we want to end up with and doesn’t involve creating a duplicate test file.